packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
```

Comment:
I think this method should be renamed to `pushOntoQueue` to avoid confusion with `Array.prototype.push`. This is especially problematic since this method is async and `Array.prototype.push` is not.

For example, the following code is an easy bug to make:

```js
this._queue.push(...);
```

It should instead be:

```js
await this._queue.push(...);
```